.github/workflows/publish.yml

```diff
@@ -30,18 +30,26 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+
+      - uses: actions/checkout@v3
```

Comment:
Checking out once is enough.

Reply:
Ehem... yeah, good point! I copied the wrong line :-D Should be fixed now.